docker-compose.yml

```
@@ -0,0 +1,85 @@
+version: '3.8'
+
+services:
+  elasticsearch6:
+    image: searchgov/elasticsearch:6.8.23
+    environment:
+      - bootstrap.memory_lock=true
+      - cluster.name=es6-docker-cluster
+      - discovery.type=single-node
+      - xpack.license.self_generated.type=trial
+      - xpack.monitoring.enabled=false
+      - xpack.security.enabled=false
+      - 'ES_JAVA_OPTS=-Xms256m -Xmx256m'
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - es_6_data:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+
+  elasticsearch7:
+    image: searchgov/elasticsearch:7.17.3
+    environment:
+      - bootstrap.memory_lock=true
+      - cluster.name=es7-docker-cluster
+      - discovery.type=single-node
+      - xpack.license.self_generated.type=trial
+      - xpack.monitoring.enabled=false:NER
```

```suggestion
      - xpack.monitoring.enabled=false
```

@MothOnMars This `:NER` (typo?) should be removed.  Upon testing the ES7 workflow on my Fearless M1 just now, I noted this throws the following on startup:

```
16T22:40:06,866Z", "level": "ERROR", "component": "o.e.b.ElasticsearchUncaughtExceptionHandler", "cluster.name": "es7-docker-cluster", "node.name": "0d3bed31bc7a", "message": "uncaught exception in thread [main]", 
"stacktrace": ["org.elasticsearch.bootstrap.StartupException: java.lang.IllegalArgumentException: Failed to parse value [false:NER] as only [true] or [false] are allowed."
```